### DISCUSSION TITLE
Spotify volume control on device not controlling volume on Player/Pi

### DISCUSSION THREAD

**Discussion by gingep** [rern/rAudio#1528] [Fixed Issues]
Hi, new user here.
I had a previous version or rAudio where changing the volume control on my mobile device running spotify would control the volume on my Pi.
This no longer happens and so I have to access rAudio.local to change the playback volume.
Any advice please.

rAudio 20231210
6.1.53-1 armv7l
Raspberry Pi 2B Rev 1.1
BCM2836 • 963 MiB
4 x Cortex-A7 @ 900 MHz

iqAudioDAC+

# aplay -l | grep ^card
card 0: IQaudIODAC [IQaudIODAC], device 0: IQaudIO DAC HiFi pcm512x-hifi-0 [IQaudIO DAC HiFi pcm512x-hifi-0]

# amixer scontrols
Simple mixer control 'DSP Program',0
Simple mixer control 'Analogue',0
Simple mixer control 'Analogue Playback Boost',0
Simple mixer control 'Auto Mute',0
Simple mixer control 'Auto Mute Mono',0
Simple mixer control 'Auto Mute Time Left',0
Simple mixer control 'Auto Mute Time Right',0
Simple mixer control 'Clock Missing Period',0
Simple mixer control 'Deemphasis',0
Simple mixer control 'Digital',0
Simple mixer control 'Max Overclock DAC',0
Simple mixer control 'Max Overclock DSP',0
Simple mixer control 'Max Overclock PLL',0
Simple mixer control 'Volume Ramp Down Emergency Rate',0
Simple mixer control 'Volume Ramp Down Emergency Step',0
Simple mixer control 'Volume Ramp Down Rate',0
Simple mixer control 'Volume Ramp Down Step',0
Simple mixer control 'Volume Ramp Up Rate',0
Simple mixer control 'Volume Ramp Up Step',0

# cat /etc/asound.conf
defaults.pcm.card 0
defaults.ctl.card 0

**Comment by HunLucas** [comment]
This isn't fixed

**Comment by AndyFx12** [comment]
see here: https://github.com/rern/rAudio/discussions/1621#discussioncomment-8773511
